x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -910,51 +910,44 @@ public void onFailure(Exception e) {
                             Throwable cause = ExceptionsHelper.unwrapCause(e);
                             if (cause instanceof VersionConflictEngineException) {
                                 //The document has been updated by another thread, get it again.
-                                if (backoff.hasNext()) {
```

Comment:
I don't think we have to back-off on version conflict. There is no danger of hammering the index as only one update will succeed. (I also did not back-off previously, but was prob an oversight because we check here if we should)